default.json

```diff
@@ -71,10 +71,11 @@
       "groupName": "golang"
     },
     {
-      "description": "Update podman container image digest weekly (rebuilt nightly)",
+      "description": "Disable pinning digest for podman container image (rebuilds nightly and keeps latest manifest only)",
```

Comment:
This is not a problem for now since it's already broken, but we want completely reproducible builds.

Reply:
Agreed, builds happen in a container within that container, so they should remain fairly reproducible. Anything that happens in the Podman container (`dnf` is used for example) or in GH Workflows in general is not guaranteed to be reproducible. So the simplest is to keep this one with tag only for now, as even without specific scheduling, Renovate always lags a little behind updates, and it could impair the workflows

I can remind you of Nix tho 😁